### ISSUE TITLE
[BUG] Search Bar/Field Problems

### ISSUE THREAD

**Issue by PCO62** [Hofknecht/SystemTrayMenu#516]
When I try **to perform a search**, in the respective field, the application links, located in the system paths, (_"ProgramData\...\StartMenu_" and _"user\AppData\..\StartMenu_", see Fig. 2), **are found (recursively)** as expected.

However, links that are in the folders (and subfolders) of the **custom directory** (Figs. 1 and 3) **are not found**, unless they are in the root folder (that is, **the search is not recursive**).

**_Bug (?!) or any/some configuration problem (?!)_**

![STM-Figs_75pc](https://github.com/Hofknecht/SystemTrayMenu/assets/93612520/3e0ebe44-f322-480b-83f6-bb67b6118de5)

 - OS: Windows11 23H2
 - Version 1.3.5.0 (UWP)

**Comment by PCO62** [comment]
I tried to make several **configuration changes** (the rational ones) and tried the **portable version of STM** (my current version is UWP), thinking that there might be some limitation and/or bug in the latter, but the result was always consistent: 
	_the recursive folder search created in the root directory cannot be performed unless the path is included in Settings > Directories (Fig. 2)_!

But, in this last situation, to have a searchable Custom Folders Menu, **we would have to do without placing system folders (Fig. 2) to avoid a mess in the Menu!**

In other words, **it is an intrinsic limitation of the App itself... which I lovef**, but which has this drawback!
_____________________________________________________
**_Will it be possible to implement this functionality in STM?_**

**This is the challenge that I, respectfully, leave to our Developer!**

**Comment by Hofknecht** [comment]
@PCO62 
Thank you for the feedback!

Performing recursive searches would require indexing the entire hard drive, which could lead to increased system load. While other programs do this, we prefer to give users control over what they need to index, ensuring the process remains fast and efficient. This way, they can specify exactly what they need to be included in the search, avoiding unnecessary system strain.

see related
#273
